### ISSUE TITLE
Make About dialog work withut JavaScript

### ISSUE THREAD

**Issue by waldyrious** [browserpad/browserpad#66]
In #65 a simple method to display a dialog box was implemented using the [dialog](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) element and its `showModal()` method.

However, it would be nice if as much of the site's functionality was available without JavaScript. So perhaps adapting one of the approaches from [this demo](https://github.github.io/details-dialog-element/example/index.html) might be a better way to have that functionality (as long as it doesn't introduce too much complexity compared with the current approach).

**Comment by SenadAlagic** [comment]
I've looked into this a bit. At first glance it would seem to be a bit more complicated than the current solution. It seems it would also still require som JS to be implemented (if I'm not mistaken). 
I'd love to hear your opinion on this :)

**Comment by waldyrious** [comment]
> It seems it would also still require som JS to be implemented (if I'm not mistaken).

It actually works with JS disabled — do try disabling JavaScript and clicking the "Dialog" button :)

The only downside seems to be that the close button only works with JS enabled, but clicking anywhere outside the dialog does appear to work, so that doesn't seem like a deal-breaker.

**Comment by SenadAlagic** [comment]
I guess I somehow took the wrong approach. I will look into this again a bit deeper.

**Comment by SenadAlagic** [comment]
I made a commit to my repo if you'd like to check it out. I implemented it the second way - the way this [this demo](https://github.github.io/details-dialog-element/example/index.html) here shows it, the first example. It doesn't really work, I'm sure I missed something. I'd be great if you could check it out. Cheers 😄

**Comment by waldyrious** [comment]
Feel free to create a draft PR with that commit — we can iterate on the implementation there :)

**Comment by SenadAlagic** [comment]
Sure thing!

**Comment by SenadAlagic** [comment]
Hello again! Since you've came back I thought I'd tackle some of these issues again. I've looked into it again and there seem to be two options:
1. Importing the details-dialog component with a link to the script 
2. Avoid importing outside code but *still* write the logic for auto closing

Clicking outside the dialog and making it close seems to be the work of this `<details-dialog>` tag.

**Comment by waldyrious** [comment]
To be clear, I don't really care if the dialog is closed via clicking outside it or pressing a button. The main goal here is to ensure the dialog still works when javascript is disabled.

**Comment by SenadAlagic** [comment]
Yes I understood that, but my point was that we would then have to link the script (not sure if you would count that as "using JavaScript"). I will definitely look into this a bit more to see how it can be done cleanly without JS 😃
